prometheus/process_collector.go

```diff
@@ -100,6 +101,16 @@ func NewProcessCollector(opts ProcessCollectorOpts) Collector {
 			"Start time of the process since unix epoch in seconds.",
 			nil, nil,
 		),
+		inBytes: NewDesc(
+			ns+"process_net_bytes_in",
+			"Number of bytes received by the process over the network.",
+			nil, nil,
+		),
+		outBytes: NewDesc(
+			ns+"process_net_bytes_out",
```

Comment:
This should follow Prometheus naming conventions.

```suggestion
			ns+"process_network_received_bytes_total",
			"Number of bytes received by the process over the network.",
			nil, nil,
		),
		outBytes: NewDesc(
			ns+"process_network_transmitted_bytes_total",
```

Reply:
Have tweaked naming, left it as in/out because that maps closer to where the actual value comes from.

Reply:
in/out does not match our naming conventions.

Reply:
Then please link me the naming convention documentation.

Reply:
https://prometheus.io/docs/practices/naming/

This one specifically is about ambiguity. In network terminology we talk about receive and transmit. For example, the node_exporter standard metrics are:

```
node_network_receive_bytes_total
node_network_transmit_bytes_total
```

We should follow that convention.